#
#

uint32 runningStatus
int32 speed
int32 position
float32 temperature
